funcions/5.js

```diff
@@ -0,0 +1,24 @@
+let text =
+	'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio necessitatibus vero voluptatum enim, maiores neque odio incidunt tempora aliquid! Nemo iusto ipsam quae quos odit soluta in nobis a doloribus?'
+
+let symbolsToRemove = /[^A-Za-z0-9]/g
+function oldWordSearch(w) {
```

Comment:
The task description probably wasn't very clear, but the idea is that a function `wordSearch` would accept a `word` and `text` arguments and would result in:
```js
wordSearch('foo', 'foobar') // false
wordSearch('foo', 'Foo bar') // true
wordSearch('foo', ' foo bar') // true
```
Consider looking into regexp with word boundary. 😉